x-pack/legacy/plugins/alerting/server/routes/create.ts

```diff
@@ -32,6 +33,9 @@ export const createAlertRoute = {
       payload: Joi.object()
         .keys({
           enabled: Joi.boolean().default(true),
+          tags: Joi.array()
+            .items(Joi.string())
+            .default([]),
```

Comment:
it's defaulted for create, but required for update.  hmmm, I kinda see how that could make sense, but is also a bit weird ...